hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -168,6 +168,8 @@
     @Deprecated
     public static final String PROP_ENABLE_JMX_DETAILED = GroupProperty.ENABLE_JMX_DETAILED.getName();
     @Deprecated
+    public static final String PROP_JMX_UPDATE_INTERVAL_SECONDS = GroupProperty.JMX_UPDATE_INTERVAL_SECONDS.getName();
+    @Deprecated
```

Comment:
Adding an already deprecated field seems odd to me. Isn't adding it to only `GroupProperty` enough?